.gitignore

```diff
@@ -1,2 +1,4 @@
 .DS_store
-.env
\ No newline at end of file
+.env
+
+static/
```

Comment:
please look up and add generic python gitignores 

`__pycache__` and other stuff shouldn't be in git, because this is generated per machine

Also don't forget to firstly delete those caches, add to gitignore and push it to the branch

Reply:
Thank you, I removed `__pycache__` and updated gitignore according to [this template](https://github.com/github/gitignore/blob/main/Python.gitignore)